cohttp-eio/src/cohttp_eio.mli

```diff
@@ -84,15 +92,14 @@ module Server : sig
   (** {1 Run Server} *)
 
   val run :
-    ?socket_backlog:int ->
-    ?domains:int ->
-    port:int ->
-    < domain_mgr : Eio.Domain_manager.t ; net : Eio.Net.t ; .. > ->
-    handler ->
-    'a
+    ?socket_backlog:int -> ?domains:int -> port:int -> 'b env -> handler -> 'a
```

Comment:
```suggestion
    ?socket_backlog:int -> ?domains:int -> port:int -> 'a env -> handler -> 'b
```
Pointless comment, but why not just using the same as below

Reply:
Done.